.github/workflows/buildkit.yml

```diff
@@ -177,10 +177,13 @@ jobs:
       -
         name: Build ${{ needs.prepare.outputs.tag }}
         run: |
-          timeout ${BUILD_TIMEOUT} ./hack/images "${{ needs.prepare.outputs.tag }}" "$IMAGE_NAME" "${{ needs.prepare.outputs.push }}"
-          if [ $? -eq 124 ]; then
+          ./hack/images "${{ needs.prepare.outputs.tag }}" "$IMAGE_NAME" "${{ needs.prepare.outputs.push }}" &
+          bpid=$!
+          sleep ${{ env.IMAGE_BUILD_TIMEOUT }}
+          if kill -0 $bpid 2>/dev/null; then
             echo "::error::Build timed out after ${BUILD_TIMEOUT} seconds"
```

Comment:
nit: fix this to `${IMAGE_BUILD_TIMEOUT}`.

Reply:
was still wip but thanks!